versionedController/v1/signup/signup.go

```diff
@@ -43,9 +43,10 @@ func (signupController *SignupController) Post(c *gin.Context) {
 	}
 
 	newUser := &models.UserCredentials{
-		UserName: signupModel.UnverifiedEmail,
-		Name:     signupModel.Name,
-		Password: signupModel.Password,
+		UserName:        signupModel.UnverifiedEmail,
+		Name:            signupModel.Name,
+		Password:        signupModel.Password,
+		UnverifiedEmail: signupModel.UnverifiedEmail,
```

Comment:
added this here so that we can verify  `UnverifiedEmail`. Else it will come as blank for the check.